### PR TITLE
Migrate prs entry script

### PR DESCRIPTION
**Pull request by jac86**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [x] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by Adrian999999999 — APPROVED** [review]

File: src/promptflow-core/promptflow/integrations/parallel_run/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from .processor import ParallelRunProcessor, create_processor
```
Comment (thy09) [inline src/promptflow-core/promptflow/integrations/parallel_run/__init__.py:4]: Since it is only used for component, maybe it should be part of pfazure?

Comment (elliotzh) [reply src/promptflow-core/promptflow/integrations/parallel_run/__init__.py:4]: current pfazure is promptflow azure devkit actually, I don't think runtime image will install it. As offline synced, we can add a new `promptflow-parallel` like [promptflow-eval](https://github.com/microsoft/promptflow/commit/078549534a3685d45cc2c8e29263c4eb6a7ca2ce) and talk to runtime team for in which image we should install this standalone package

Comment (jac86) [reply src/promptflow-core/promptflow/integrations/parallel_run/__init__.py]: moved parallel processing code to promptflow-parallel

File: .github/workflows/promptflow-parallel-e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      - src/promptflow-parallel/**
+      - .github/workflows/promptflow-parallel-e2e-test.yml
+  workflow_dispatch:
+
+env:
+  IS_IN_CI_PIPELINE: "true"
+  WORKING_DIRECTORY: ${{ github.workspace }}/src/promptflow-parallel
+
+jobs:
+  build:
```
Comment (crazygao) [inline .github/workflows/promptflow-parallel-e2e-test.yml:20]: for daily run it is not necessary to make these steps.
Please refer to .github/workflows/promptflow-sdk-cli-test.yml and simplyfy

By the way I didn't see any test need real aoai connections.
All the azure login steps could be omitted.

File: src/promptflow/setup.py [hunk 3]
```diff
@@ -54,6 +54,7 @@
             "promptflow-core[executor-service]",
             "promptflow-devkit[all]",
             "promptflow-azure",
+            "promptflow-parallel",
```
Comment (crazygao) [inline src/promptflow/setup.py:57]: put it here means that this is mandatory parts of promptflow.

File: src/promptflow-parallel/pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+]
+
+[tool.poetry.urls]
+"Bug Reports" = "https://github.com/microsoft/promptflow/issues"
+
+[tool.poetry.dependencies]
+python = "<4.0,>=3.8"
+promptflow-core = "<2.0.0"
+
+[tool.poetry.group.test.dependencies]
```
Comment (crazygao) [inline src/promptflow-parallel/pyproject.toml:42]: refer to devkit repo, this file has some updates.

File: src/promptflow-parallel/tests/parallel/.coveragerc [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+    *__init__.py*
```
Comment (crazygao) [inline src/promptflow-parallel/tests/parallel/.coveragerc:3]: This file is not necessary,

Comment (jac86) [reply src/promptflow-parallel/tests/parallel/.coveragerc:3]: removed

File: src/promptflow/setup.py (hunk 3 above)
Comment (jac86) [reply src/promptflow/setup.py]: as discussed offline, it can be removed

File: .github/workflows/promptflow-parallel-e2e-test.yml [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  IS_IN_CI_PIPELINE: "true"
+  WORKING_DIRECTORY: ${{ github.workspace }}/src/promptflow-parallel
+
+jobs:
+  test:
```
Comment (crazygao) [inline .github/workflows/promptflow-parallel-e2e-test.yml:24]: create a special name for each job name.
This is important handler for github.

Comment (crazygao) [reply .github/workflows/promptflow-parallel-e2e-test.yml:24]: also the other one.

File: src/promptflow-parallel/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Prompt flow parallel computing
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-evals)](https://pypi.org/project/promptflow-evals/)
```
Comment (wangchao1230) [inline src/promptflow-parallel/README.md:3]: need to update the link here

File: src/promptflow-parallel/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+[![Python](https://img.shields.io/pypi/pyversions/promptflow.svg?maxAge=2592000)](https://pypi.python.org/pypi/promptflow-core/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+# Introduction
+
```
Comment (wangchao1230) [inline src/promptflow-parallel/README.md:8]: add one line introduction here

File: src/promptflow-parallel/pyproject.toml [hunk 9]
```diff
@@ -0,0 +1,106 @@
+[tool.poetry]
+name = "promptflow-parallel"
+version = "1.10.0.dev0"
```
Comment (wangchao1230) [inline src/promptflow-parallel/pyproject.toml:3]: may start with some 0.x version, no need to align with promptlfow package?

File: .github/workflows/promptflow-parallel-e2e-test.yml (hunk 6 above)
Comment (jac86) [reply .github/workflows/promptflow-parallel-e2e-test.yml]: renamed

File: src/promptflow-parallel/README.md (hunk 7 above)
Comment (jac86) [reply src/promptflow-parallel/README.md]: updated

File: src/promptflow-parallel/README.md (hunk 8 above)
Comment (jac86) [reply src/promptflow-parallel/README.md]: updated

File: src/promptflow-parallel/pyproject.toml (hunk 9 above)
Comment (jac86) [reply src/promptflow-parallel/pyproject.toml]: changed to 0.1.0.dev0

**Review by crazygao — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by thy09 — APPROVED** [review]

**Pull request merged**